src/aiidalab_qe/common/widgets.py

```diff
@@ -430,6 +432,7 @@ class AddingTagsEditor(ipw.VBox):
     structure = traitlets.Instance(ase.Atoms, allow_none=True)
     selection = traitlets.List(traitlets.Int, allow_none=True)
     input_selection = traitlets.List(traitlets.Int, allow_none=True)
+    structure_node = traitlets.Instance(orm_Data, allow_none=True, read_only=True)
```

Comment:
```suggestion
    structure_node = traitlets.Instance(orm.StructureData, allow_none=True, read_only=True)
```
This is a structure data, no?